### ISSUE TITLE
MinGW 32-bit Debug: Example application PyScriptingConsole_d.exe crashes on start

### ISSUE THREAD

**Issue by githubuser0xFFFF** [MeVisLab/pythonqt#21]
Hi,

I compiled PythonQt with MinGW 7.3.0 64-bit and MinGW 7.3.0 32-bit for debug and release. Before I did this, I installed the additional Python debug binaries (python38_d.dll) via the Python installer.

The provided example applications work properly for MinGW 64-bit debug and release and for MinGW 32-bit release. The MinGW 32-bit debug build causes a crash of several examples. The following examples crash immediatelly on start:

```
PyScriptingConsole_d.exe
PyLauncher_d.exe
PyGuiExample_d.exe
```

The applications crash before the main or qMain is entered. That means, even If I debug the applications in QtCreator I cannot set a breakpoint see any aditional information. The only information I see is a Windows message box telling me:

`The application was unable to start correctly (0xc0000017). Click OK to cloase the application`

Is this something someone has seen before? It is realy strange that this happens only for the debug version of the MinGW 32-bit build and only for the three examples.

**Environment:**
Qt: 5.12.2
Python: 3.8.2 32-bit and 64-bit
MinGW: 7.3.0

**Comment by mrbean-bremen** [comment]
@githubuser0xFFFF - is this still an issue, or has this been fixed with #20? (going over old issues)

**Comment by githubuser0xFFFF** [comment]
No, this is no issue anymore because we use Mingw 11 now and do not link to python debug DLL anymore. You can close the issue.
